bears/php/PHPCodeSnifferBear.py

```diff
@@ -0,0 +1,47 @@
+import tempfile
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='phpcs',
+        output_format='regex',
+        output_regex=r'(?P<filename>.+):(?P<line>\d+):(?P<column>\d+): '
+                      '(\w+) - (?P<message>.+)')
+class PHPCodeSnifferBear:
+    """
+    Tokenises and "sniffs" PHP, JavaScript and CSS files to detect violations
+    of a defined coding standard.
+
+    See <http://pear.php.net/manual/en/package.php.php-codesniffer.php> for
+    more information.
+    """
+
+    LANGUAGES = {"PHP", "JavaScript", "CSS"}
+    REQUIREMENTS = {DistributionRequirement(apt_get='php-codesniffer')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax', 'Documentation',
+                  'Code Simplification'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         max_line_length: int=79):
+        configs = '''<?xml version="1.0"?>
+<ruleset name="Custom Standard">
+ <description>A custom coding standard</description>
+<rule ref="PEAR"/>
+ <rule ref="Generic.Files.LineLength">
+  <properties>
+   <property name="lineLimit" value="{max_line_length}"/>
+  </properties>
+ </rule>
+</ruleset>
+'''.format(max_line_length=max_line_length)
```

Comment:
why don't you use linter's generate config functionality that generates a tempfile for you?